lib/ios/UITabBarController+RNNOptions.m

```diff
@@ -74,6 +75,12 @@ - (void)setTabBarVisible:(BOOL)visible animated:(BOOL)animated {
 	}
 }
 
+- (void)setTabItemImagesCentered:(BOOL)centered {
```

Comment:
This setter doesn't set the value, instead it hides an imperative function - seems redundant as it doesn't handle the other case when `centered` is false.